When building the dbus-ptest package, we have to enable python.  However
checking if the host-system python has the necessary library isn't useful.

Disable the python module check for cross compiling.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

--- dbus-1.6.8/configure.ac.orig	2013-07-11 14:15:58.834554799 -0500
+++ dbus-1.6.8/configure.ac	2013-07-11 14:14:40.969554848 -0500
@@ -257,13 +257,6 @@
   # full test coverage is required, Python is a hard dependency
   AC_MSG_NOTICE([Full test coverage (--enable-tests=yes) requires Python, dbus-python, pygobject])
   AM_PATH_PYTHON([2.6])
-  AC_MSG_CHECKING([for Python modules for full test coverage])
-  if "$PYTHON" -c "import dbus, gobject, dbus.mainloop.glib"; then
-    AC_MSG_RESULT([yes])
-  else
-    AC_MSG_RESULT([no])
-    AC_MSG_ERROR([cannot import dbus, gobject, dbus.mainloop.glib Python modules])
-  fi
 else
   # --enable-tests not given: do not abort if Python is missing
   AM_PATH_PYTHON([2.6], [], [:])
